homeassistant/components/vicare/binary_sensor.py

```diff
@@ -145,34 +148,37 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Create the ViCare binary sensor devices."""
-    api = hass.data[DOMAIN][config_entry.entry_id][VICARE_API]
+    entities: list[ViCareBinarySensor] = []
+
+    for device_config, api in hass.data[DOMAIN][config_entry.entry_id][
+        DEVICE_CONFIG_LIST
+    ]:
+        if device_config.getModel() == "Heatbox1":
```

Comment:
With this we ignore the Gateway device which is currently wrongly detected as *generic heating device*.